src/NCronJob/Configuration/Builder/NCronJobOptionBuilder.cs

```diff
@@ -143,7 +143,7 @@ internal static CronExpression GetCronExpression(string expression)
             : throw new InvalidOperationException("Invalid cron expression");
     }
 
-    private (JobOptionBuilder, IReadOnlyCollection<JobDefinition>) AddJobInternal(
+    private List<JobDefinition> AddJobInternal(
```

Comment:
Changed from `IReadOnlyCollection<T>` to `List<T>` to please [CA1859](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1859) warning.

Reply:
Ahh yeah - absolutely fair. THanks for the hint